src/cockpit/packages.py

```diff
@@ -324,7 +324,6 @@ def serve_file(self, path, channel):
             data, (content_type, encoding) = self.files[path]
 
         headers = {
-            "Access-Control-Allow-Origin": channel.origin,
```

Comment:
This was done 6 years ago in commit e5f1b66db00d. I have some feeling that this was done for embedding cockpit frames into other pages. But (1) we at least have check-embed, and (2) we essentially deliberately disallowed cross-origin frame inclusion commit 9a45328950, see #16122. (Also, I keep forgetting the details of CORS, it's too complex for my brain :exploding_head: )

So, I think commit 9a45328950 effectively just forgot to clean up these headers in the C bridge, the Py bridge just copied it, and I agree this can all go. :+1: